# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1356123926 18000
#      Fri Dec 21 16:05:26 2012 -0500
# Branch stable
# Node ID 23a180fd186d8e7f80c248119ac6daee66aa950c
# Parent  ebe8c29644514d2b416e8dfff1efbbdbabdab360
Remove doc-cache file in maintainer-clean target, not distclean

* doc/interpreter/Makefile.am (MAINTAINERCLEANFILES):
List doc-cache here.
(DISTCLEANFILES): Not here.

diff --git a/doc/interpreter/Makefile.am b/doc/interpreter/Makefile.am
--- a/doc/interpreter/Makefile.am
+++ b/doc/interpreter/Makefile.am
@@ -236,29 +236,29 @@ spellcheck: $(SPELLCHECK_FILES)
 	else \
 		echo "Spellcheck passed"; \
 	fi
 .PHONY: spellcheck
 
 EXTRA_DIST = \
   config-images.sh \
   contributors.in \
-	doc-cache \
+  doc-cache \
   find-docstring-files.sh \
   images \
   images.awk \
   images.mk \
   mk_doc_cache.m \
   mkcontrib.awk \
   munge-texi.pl \
   octave.dvi \
   octave.html \
   octave.pdf \
   octave.ps \
   $(IMAGES) \
   $(IMAGES_SRC) \
   $(LOGOS) \
   $(TXI_SRC)
 
-DISTCLEANFILES = $(octave_TEXINFOS) doc-cache
+DISTCLEANFILES = $(octave_TEXINFOS)
 
-MAINTAINERCLEANFILES = $(IMAGES)
+MAINTAINERCLEANFILES = $(IMAGES) doc-cache
 
